src/cli/commands/fellowship/createGame.ts

```diff
@@ -0,0 +1,101 @@
+import { Command } from "commander";
```

Comment:
CamelCase in cli file names suggests that files is in wrong location. 

I would suggest moving it under `src/cli/commands/fellowship/threads/create.ts